### PR TITLE
Default sh-select issue fixed

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 1]
```diff
@@ -289,6 +290,9 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
         // });
         this.store.pipe(select(prof => prof.settings.profile), takeUntil(this.destroyed$)).subscribe((profile) => {
             this.inputMaskFormat = profile.balanceDisplayFormat ? profile.balanceDisplayFormat.toLowerCase() : '';
+            if(profile && profile.countryV2 && profile.countryV2.alpha2CountryCode) {
+            this.isIndianCompany = profile.countryV2.alpha2CountryCode === 'IN'? true: false;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:294]: format

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 2]
```diff
@@ -289,6 +290,9 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
         // });
         this.store.pipe(select(prof => prof.settings.profile), takeUntil(this.destroyed$)).subscribe((profile) => {
             this.inputMaskFormat = profile.balanceDisplayFormat ? profile.balanceDisplayFormat.toLowerCase() : '';
+            if(profile && profile.countryV2 && profile.countryV2.alpha2CountryCode) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:293]: Format code

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 3]
```diff
@@ -83,6 +83,7 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
     public addBankForm: FormGroup;
     /** Input mast for number format */
     public inputMaskFormat: string = '';
+    public isIndianCompany: boolean = true;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:86]: Missing JS doc

File: apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html [hunk 4]
```diff
@@ -12,8 +12,8 @@ <h4 class="modal-title pull-left">Payment</h4>
                     <div class="d-flex mb-1">
                         <div class="form-group mr-15 select-bank">
                             <label>Select Bank<span class="red-text">*</span></label>
-                            <sh-select [options]="selectIntegratedBankList" [(ngModel)]="selectedBankUniqueName" [fixedValue]="selectedBankName" [defaultValue]="selectedBankName" class="text-left" name="selectedBankUniqueName" required placeholder="'Select Bank'" (click)="setBankName($event);"
-                                (selected)="selectBank($event);" [multiple]="false">
+                            <sh-select [options]="selectIntegratedBankList" #selectBankEle [multiple]="false" [ItemHeight]="33" [fixedValue]="selectedBankName" [isFilterEnabled]="true" class="text-left" [(ngModel)]="selectedBankUniqueName" name="selectedBankUniqueName" required
+                                placeholder="'Select Bank'" (click)="setBankName($event, selectBankEle);" (selected)="selectBank($event);" [multiple]="false">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html:16]: Remove inner '' in placeholder attribute

**Pull request merged**